src/components/DialogConfigButton.js

```diff
@@ -0,0 +1,124 @@
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Typography,
+  TextField,
+  Box
+} from '@mui/material'
+import SettingsIcon from '@mui/icons-material/Settings'
+import React, { useState } from 'react'
+import {
+  createChromeAlarm,
+  INITIAL_CHECK_INTERVAL_TIMER_MINUTES
+} from '../background'
+import { useChromeLocalStorage } from '../hooks/useChromeLocalStorage'
+
+const SETTINGS_LABELS = {
+  checkIntervalTimer: 'Check Interval Timer (minutes)'
+}
+
+const rowStyles = {
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-around'
+}
+
+const ConfigurationTitle = ({ title }) => {
+  return (
+    <Box sx={{ maxWidth: '50%' }}>
```

Comment:
IMO as the `sx` property never change, you can extract its value as a `const` outside of the component in order to avoid useless re-render

Reply:
I don't see how the inline styles would cause useless re-render. Could you please clarify?

Reply:
Each time that the title changes, here you are creating a new object.
Since each new object has a different reference, the re-render is triggered

Reply:
although strictly speaking @ilteoood is right, in practice there will be a rerender of something anyway, since title is changing, so for the sake of simplicity I'd keep the styles inline